docs/vgclog.md

```diff
@@ -0,0 +1,423 @@
+<!--
+* Copyright (c) 2017, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Verbose Garbage Collection Logs 
+
+[Garbage collection](gc.md)(GC) reclaims used memory in the heap for reuse. During cleanup of the heap, the verbose garbage collection logs capture information about the different garbage collection operations that are involved in the garbage collection cycles. GC operations include GC initialization, stop-the-world processing, finalization, reference processing, and allocation failures. 
```

Comment:
> GC operations include GC initialization, stop-the-world processing, finalization, reference processing, and allocation failures. 

Not sure what was the goal of this sentence, but it is confusing for a few reasons....

GC operations are parts of a cycle (as already stated in previous sentence) that accomplish specific functional goal toward reclaiming memory, such as 1) traversal of live object graph and marking or moving those objects, 2) sweeping memory that contain dead objects, 3) class unloading, 4) heap compaction...

Initialization is not really a GC operation (it's not a part of a cycle, and does not contribute toward memory reclamation). 

STW processing is not a GC operation, it's just one way to perform (concurrent being the other one) a certain operation or an increment of an operation.

Finalization and reference processing are not really separate GC operations, purely from Verbose GC point of view (no gc-op tag for them). In some sense,  yes they are GC operations and possibly could be reported as separate gc-ops but they are not. I would not mention them here.

Allocation failure is not an operation, it's an event that triggers a cycle start or an increment.

Reply:
Taking this sentence out - it was an amalgamation of incorrect legacy information.